src/openrct2-ui/windows/Park.cpp

```diff
@@ -1565,8 +1566,19 @@ static void window_park_objective_paint(rct_window *w, rct_drawpixelinfo *dpi)
             gfx_draw_string_left_wrapped(dpi, nullptr, x, y, 222, STR_OBJECTIVE_FAILED, COLOUR_BLACK);
         } else {
             // Objective completed
-            set_format_arg(0, money32, gScenarioCompletedCompanyValue);
-            gfx_draw_string_left_wrapped(dpi, gCommonFormatArgs, x, y, 222, STR_OBJECTIVE_ACHIEVED, COLOUR_BLACK);
+            if ((gConfigGeneral.allow_early_completion || gScenarioObjectiveType == OBJECTIVE_10_ROLLERCOASTERS ||
```

Comment:
I think this information should just be kept to situations that have early completion on.

Reply:
@AaronVanGeffen Also made this comment, and I pointed out that these scenario types are the ones that can be speedrun in the traditional sense. (See march 22 comments above)  I would be fine with restricting it to only show when early completion is enabled if that's what you decide should be there

Reply:
I think this would be useful in non early-completion as well to be honest.

Reply:
What I meant to say was that I could see this being useful in scenarios that aren't the 10 coasters as well. Basically, Display it if early completion is on, reguardless of the scenario objective type.